src/gateways/internal-api.ts

```diff
@@ -13,14 +15,39 @@ export class InternalServerError extends Error {
   }
 }
 
+export interface EvidenceRequestRequest {
+  deliveryMethods: string[];
+  documentTypes: string[];
+  resident: Omit<IResident, 'id'>;
+  serviceRequestedBy?: string;
+}
+
 export class InternalApiGateway {
   async getEvidenceRequests(): Promise<EvidenceRequest[]> {
+    await new Promise((resolve) => setTimeout(resolve, 100));
+    return EvidenceRequestsFixture.map(ResponseMapper.mapEvidenceRequest);
+    // try {
+    //   const { data } = await Axios.get<EvidenceRequestResponse[]>(
+    //     '/api/evidence/evidence_requests'
+    //   );
+
+    //   return data.map((er) => ResponseMapper.mapEvidenceRequest(er));
+    // } catch (err) {
+    //   console.log(err);
+    //   throw new InternalServerError('Internal server error');
+    // }
```

Comment:
Should this comment be removed?

Reply:
It'll be needed once the endpoint is done and this can be uncommented